app/routes/objects.js

```diff
@@ -0,0 +1,39 @@
+import Joi from 'joi'
+import { StatusCodes } from 'http-status-codes'
+
+import { handleObjectRetrieval } from '../services/retrieve.js'
+import { FILE_NOT_FOUND } from '../constants/file-errors.js'
+
+const handleRetrievalError = (err, h) => {
+  switch (err?.cause) {
```

Comment:
Sonarcloud is flagging this to be refactored to an `if` statement so I would update it